lib/iris/fileformats/rules.py

```diff
@@ -121,7 +120,6 @@ def _rules_execution_environment():
         _rules_globals.update(iris.coords.__dict__)
         _rules_globals.update(iris.coord_systems.__dict__)
         _rules_globals.update(iris.fileformats.um_cf_map.__dict__)
-        _rules_globals.update(cf_units.__dict__)
```

Comment:
It isn't.
I got confused...
I will fix that + force-upload it, to avoid having that code reversion in the branch...